website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,317 @@
+# Operator Scope
+
+## Overview
+
+A namespace-scoped operator watches and manages resources in a single namespace, whereas a cluster-scoped operator 
+watches and manages resources cluster-wide.
+
+A cluster-scoped scope is ideal for operator projects which will control resources in more than one namespace. 
+At the same time, namespace-scoped operators are preferred because of their flexibility. They enable decoupled upgrades, 
+namespace isolation for failures and monitoring, and differing API definitions.
+
+By default, `operator-sdk init` scaffolds a cluster-scoped operator. This document details conversion of default 
+operator projects to namespaced-scoped operators. However, there are use cases where a cluster-scoped operator may 
+make sense. For example, the [cert-manager][cert-manager] operator is often deployed with cluster-scoped permissions 
+and watches so that it can manage and issue certificates for an entire cluster.
+
+**IMPORTANT**: When a [Manager][ctrl-manager] instance is created in the `main.go` file, it might receives the 
+namespace(s) as Options as described below. These namespace(s) should be watched and cached for the Client which is 
+provided by the Controllers. Only clients provided by cluster-scoped projects are able to manage cluster-scoped CRD's. 
+For further information see: [CRD scope doc][crd-scope-doc].
+
+## Watching and Managing resources in all cluster (Default)
+
+A [Manager][ctrl-manager] is responsible for managing the controllers of the operator project. In the `main.go` you will 
+be able to find it. See:
+
+```go
+...
+mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+    Scheme:             scheme,
+    MetricsBindAddress: metricsAddr,
+    Port:               9443,
+    LeaderElection:     enableLeaderElection, 
+    LeaderElectionID:   "f1c5ece8.example.com",
+})
+if err != nil {
+    setupLog.Error(err, "unable to start manager")
+    os.Exit(1)
+}
+...
+```
+
+**NOTE** The above example is the default code generated for the projects created with `operator-sdk init` which is 
+creating a new Manager for an operator `cluster-scoped`.
+
+## Watching and Managing resources in an Namespace
+
+To restrict the scope of the [Manager's][ctrl-manager] cache to an specific `namespace` is required to inform it via 
+the `Namespace` attribute from [Options][ctrl-options]. See: 
```

Comment:
Why highlight "namespace"? We're not doing it anywhere else.
Plus we've already linked to the Manager godocs enough times above.
```suggestion
To restrict the scope of the Manager's cache to a specific namespace set the Namespace option:
```

Reply:
addressed with a few caveats.